### PR TITLE
edits to market framework

### PR DESCRIPTION
**Pull request by tamlyn10**

### CODE REVIEW

**Comment by tamlyn10** [comment]
We should probably update the market framework to more clearly define the taxonomy, as per: https://vegaprotocol.slack.com/archives/C9E6XA8GK/p1603108441178600

cc @barnabee

File: specs/0001-market-framework.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ Data:
   - **Mark price methodology parameters:**
     - Algorithm 1 / Last Traded Price: initial mark price
   - **Price monitoring parameters**: a list of parameters, each specifying one price monitoring auction trigger and the associated auction duration.
-
+  - **Default trading commences**: datetime submitted with the market creation proposal, this will affect the length of the opening auction. 
+  - **Default trading terminated**: datetime submitted with the market creation proposal, this will determine when trading ceases (prior to settlement).
```
Comment (witgaw) [inline specs/0001-market-framework.md]: Should we put them both in the same tense? So "Default trading terminates" I guess.

File: specs/0001-market-framework.md [hunk 2]
```diff
@@ -198,15 +199,24 @@ Market {
             metadata: InstrumentMetadata {
                 tags: [
                     "asset_class:fx/crypto",
-                    "product:futures"
+                    "product:futures",
+                    "underlying:BTC/USD",
+                    "fx/base: BTC",
```
Comment (witgaw) [inline specs/0001-market-framework.md:209]: Ashley mentioned the other day on slack that "base" / "quote" fields are getting removed, but I don't know the details, assumed it was a spec change?

**Review by witgaw — APPROVED** [review]
Two minor comments, looks good to me otherwise.

File: specs/0001-market-framework.md (hunk 1 above)
Comment (tamlyn10) [reply specs/0001-market-framework.md]: Adjust trading terminated to be on the Product.. time-based-expiry products. Emitting a signal that ends the instruments' trading.

File: specs/0001-market-framework.md [hunk 3]
```diff
@@ -100,17 +101,17 @@ Products must expose certain data to Vega WHEN they are instantiated as an instr
 - **Settlement assets:** one or more  assets that can be involved in settlement
 - **Margin assets:** one or more  assets that may be required as margin (usually the same set as settlement assets, but not always)
 - **Price / quote units:** the unit in which prices (e.g. on the order book are quoted), usually but not always one of the settlement assets. Usually but not always (e.g. for bonds traded on yield, units = % return or options traded on implied volatility, units = % annualised vol) an asset (currency, commodity, etc.)
-- **Status:** e.g. Active | Matured (these are the only statuses I can think of for now)
+- **Status:** e.g. Proposed | Active | Closed | Cancelled (see [market lifecycle spec]())
```
Comment (tamlyn10) [inline specs/0001-market-framework.md]: Move this to the market level...

**Pull request merged**